src/config/configManager.go

```diff
@@ -208,10 +210,11 @@ func LoadConfigFromFile() {
 
 	// recommend policy configurations
 	CurrentCfg.ConfigRecommendPolicy = types.ConfigRecommendPolicy{
-		CronJobTimeInterval:     "@every " + viper.GetString("recommend.cron-job-time-interval"),
-		OneTimeJobTimeSelection: "", // e.g., 2021-01-20 07:00:23|2021-01-20 07:00:25
-		OperationMode:           viper.GetInt("recommend.operation-mode"),
-		RecommendHostPolicy:     viper.GetBool("recommend.recommend-host-policy"),
+		CronJobTimeInterval:                "@every " + viper.GetString("recommend.cron-job-time-interval"),
+		OneTimeJobTimeSelection:            "", // e.g., 2021-01-20 07:00:23|2021-01-20 07:00:25
+		OperationMode:                      viper.GetInt("recommend.operation-mode"),
+		RecommendHostPolicy:                viper.GetBool("recommend.recommend-host-policy"),
+		RecommendAdmissionControllerPolicy: viper.GetBool("recommend.recommend-admission-controller-policy"),
```

Comment:
```suggestion
		RecommendAdmissionControllerPolicy: viper.GetBool("recommend.admission-controller-policy"),
```
can we us simplified config name?

Reply:
Done, changed both `host-policy` and `admission-controller-policy`